templates/apollo-starter-ts/model/Types.graphql

```diff
@@ -0,0 +1,14 @@
+type Note {
```

Comment:
I would prefer for templates to not have this embedded. 
We going to introduce the concept of example schemas. 
For the moment it is best to just have those inside cmd and copy that after.
We can also provide a parameter for people to init command with their own schema later.